Emby.Server.Implementations/Library/UserManager.cs

```diff
@@ -888,7 +888,7 @@ private async Task<PasswordPinCreationResult> CreatePasswordResetPin()
 
             text.AppendLine("Use your web browser to visit:");
             text.AppendLine(string.Empty);
-            text.AppendLine(localAddress + "/web/forgotpasswordpin.html");
+            text.AppendLine(localAddress + "/web/index.html#!/forgotpasswordpin.html");
```

Comment:
Can you explain this black magic? I presume it has something to do with JF trying to be a one-page app...

Reply:
I think the html file is more like a template/subpage that the index.html "app" is supposed to pull in. To point directly at it doesn't load any of the surrounding javascript and renders some useless html.

Reply:
Makes sense, thanks for enlightening me!